src/components/events/eventChecklist.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+import { List, Checkbox, Header, Segment, Button, Form } from 'semantic-ui-react';
+
+class EventChecklist extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checklist: [],
+      addingTask: false,
+      isFetching: false,
+      newTask: "",
+      submittingTask: false,
+    }
+    this.handleToggle = this.handleToggle.bind(this);
+    this.toggleTaskForm = this.toggleTaskForm.bind(this);
+    this.onTaskChange = this.onTaskChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.deleteTask = this.deleteTask.bind(this);
+  }
+  componentDidMount() {
+    this.fetchChecklist();
+  }
+  componentDidUpdate(prevProps) {
+    if (prevProps.groupEventId !== this.props.groupEventId) {
+      this.fetchChecklist();
+    }
+  }
+
+  async fetchChecklist() {
+    this.setState({ isFetching: true });
```

Comment:
lo mismo que en el dashboard, creo que es mejor que esta variable este seteada a true en el this.state, y luego cambiarla una sola vez a false luego de que termina el DidMount

Reply:
funciona también